src/page-objects/administration/Categories.ts

```diff
@@ -0,0 +1,25 @@
+import type { Page, Locator } from '@playwright/test';
+import type { PageObject } from '../../types/PageObject';
+
+export class Categories implements PageObject {
+
+    /**
+     * Landing pages
+     */
+    public readonly landingPageArea: Locator;
+    public readonly landingPageHeadline: Locator;
+    public readonly addLandingPageButton: Locator;
+    public readonly landingPageItems: Locator;
+
+    constructor(public readonly page: Page) {
+
+        this.landingPageArea = page.locator('.sw-category-detail__landing-page-collapse');
+        this.landingPageHeadline = this.landingPageArea.locator('.sw-category-detail__collapse-headline');
+        this.addLandingPageButton = this.landingPageArea.locator('.sw-landing-page-tree__add-button-button');
```

Comment:
I would recommend using the following locator:

`this.landingPageArea.getByRole( 'button', { name: 'Add landing page' });`

We try to use as much as possible the recommended built-in locators from playwright: [see here](https://playwright.dev/docs/locators) ...if that is not possible then of course the `.locator` method